src/cpp/src/cache_manager.hpp

```diff
@@ -104,13 +54,13 @@ class CacheManager {
                 ov::PartialShape pshape;
 
                 if (name.find("key_cache.") == 0) {
-                    pshape = to_partial_shape(kv_cache_config[kv_input_index], cache_precision, true);
+                    pshape = input.get_partial_shape();
                     m_block_size_in_bytes += pshape[1].get_length() * pshape[2].get_length() * pshape[3].get_length() * cache_precision.size();
```

Comment:
please, apply this diff to fix CB unit tests:
```
$ git diff
diff --git a/tests/cpp/helper.cpp b/tests/cpp/helper.cpp
index da242da4..ec3ab510 100644
--- a/tests/cpp/helper.cpp
+++ b/tests/cpp/helper.cpp
@@ -10,6 +10,10 @@ std::shared_ptr<ov::Model> get_dummy_model(ov::Core core, size_t num_layers) {
     ov::element::Type kv_cache_type = core.get_property("CPU", ov::hint::kv_cache_precision);

     auto shape = ov::PartialShape::dynamic(4);
+    shape[1] = 12;
+    shape[2] = 64;
+    shape[3] = 64;
+
     for (size_t i = 0; i < num_layers; i++) {
         auto key = std::make_shared<ov::op::v0::Parameter>(kv_cache_type, shape);
         auto value = std::make_shared<ov::op::v0::Parameter>(kv_cache_type, shape);
```

Reply:
@ilya-lavrenov , applied, thanks!